libs/sdk-react-native/ios/RNBreezSDK.m

```diff
@@ -195,6 +195,24 @@ @interface RCT_EXTERN_MODULE(RNBreezSDK, RCTEventEmitter)
     rejecter: (RCTPromiseRejectBlock)reject
 )
 
+RCT_EXTERN_METHOD(
+    fetchReverseSwapFees: (RCTPromiseResolveBlock)resolve
+    rejecter: (RCTPromiseRejectBlock)reject
+)
+
+RCT_EXTERN_METHOD(
+    inProgressReverseSwaps: (RCTPromiseResolveBlock)resolve
+    rejecter: (RCTPromiseRejectBlock)reject
+)
+
+RCT_EXTERN_METHOD(
+    sendOnchain: (NSInteger*)amountSat
+    onchainRecipientAddress: (NSString*)onchainRecipientAddress
+    pairHash: (NSString*)pairHash
+    satPerVbyte: (NSInteger*)satPerVbyte
```

Comment:
These `NSInteger` types should be `NSUInteger` here in `ios/RNBreezSDK.m`. In the swift implementation you can use the `UInt64` type for the same params.